### DISCUSSION TITLE
Need help with my project

### DISCUSSION THREAD

**Discussion by Smor3-dev** [community/community#68808] [Programming Help]
### Body

Idk why isnt loading the style.css and the others .html

When it open on the VSC its ok but when i upload it on github just broke. 
Some help would be great.

Repository: https://github.com/santisapee/Santicreatividad
Page: https://santisapee.github.io/Santicreatividad/

### Guidelines

- [X] I have read and understood this category's [guidelines](https://github.com/orgs/community/discussions/51384) before making this post.

**Comment by vishalkrishna8** [comment]
Hello @santisapee 
Here are some common issues and troubleshooting steps:
* Check file paths and names.
* Use relative paths.
* Verify file extensions.
* Match GitHub structure to your local project.
* Ensure the correct branch for GitHub Pages.
* Clear browser cache.
* Watch for case sensitivity.
* Look for 404 errors in the browser console.
* Check GitHub Actions if used.
* Confirm external resources are accessible.
* Avoid mixed content (use HTTPS).
* Check GitHub's status page for issues.

**Comment by Smor3-dev** [comment]
[SOLVED] just needed to add ../ to the path.

**Comment by MadhushaPrasad** [comment]
I think now your website deployment is working. Right??

**Reply by Smor3-dev** [reply]
yes
just added ../ on the other html, on index just need to be simple. only /
